ACTIONS.md

```diff
@@ -0,0 +1,212 @@
+# Workflows
+The following are orquesta workflow actions that are currently implemented.
+
+NOTE: We have decided not to use orquesta going forward - mainly due to the difficulty with testing and
+recovery - these workflows will be deprecated in favor of actions that do the same thing.
+
+If you're a user, this won't affect you - we'll ensure that an action is created to replace any workflow we deprecate .
+
+
+## Project workflows
+workflows made to create pre-configured Openstack projects
+
+
+| Workflow Name           | Description                                                                                           |
+|-------------------------|-------------------------------------------------------------------------------------------------------|
+| project.create.internal | Orquesta workflow to create a pre-configured Openstack Project to be used internally                  |
+| project.create.external | Orquesta workflow to create a pre-configured Openstack project to be made available externally <br/>  |
+
+
+## Check Workflow
+
+workflows made to check certain things in OpenStack
+
+| Workflow Name                    | Description                                                   |
+|----------------------------------|---------------------------------------------------------------|
+| workflow.checks.old.snapshots    | Checks for snapshots that were last updated before this month |
+| workflow.checks.security.groups  | Checks for security groups that meet the given parameters     |
+
+
+## Email Workflows (Deprecated)
+
+The following are email workflows that are deprecated and need to be rewritten to use the
+Query Library
+
+| Workflow Name              | Description                                                                                 |
+|----------------------------|---------------------------------------------------------------------------------------------|
+| email.errored.server.users | Executes the email.server.users action for errored VMs to informing users about them        |
+| email.shutoff.server.users | Executes the email.server.users action for shutoff VMs to informing users about them        |
+| email.stale.server.users   | Executes the email.server.users action for VMs with an updated timestamp older than 90 days |
+
+
+# Actions
+
+Generally, Stackstorm openstack actions are written to be very similar and easy to understand for anyone who has used
+Openstack CMD commands
+
+- each openstack resource `server, hypervisor, float.ip, etc` (should) have 4 basic commands:
+  - `create`, `delete`, `update`, `show`
```

Comment:
I don't think we want to set this as a policy, some of them are really difficult to create, might not be safe to delete or might be immutable

Reply:
removing